src/gui/processing/models/qgsmodelgraphicsscene.cpp

```diff
@@ -60,6 +60,36 @@ void QgsModelGraphicsScene::mousePressEvent( QGraphicsSceneMouseEvent *event )
   QGraphicsScene::mousePressEvent( event );
 }
 
+void QgsModelGraphicsScene::updateBounds()
+{
+  setSceneRect( modelBounds( 50 ) );
```

Comment:
can you make the 50 magic value a `static constexpr int SCENE_COMPONENT_MARGIN`?